### PR TITLE
[#1671] Upload files to subfolders

### PR DESCRIPTION
**Pull request by mauriel133**
The work in this PR allows for files to be uploaded to the current directory in the resource landing page file browser.

### CODE REVIEW

File: hs_core/views/__init__.py [hunk 1]
```diff
@@ -99,6 +99,12 @@ def add_files_to_resource(request, shortkey, *args, **kwargs):
     res_files = request.FILES.values()
     extract_metadata = request.REQUEST.get('extract-metadata', 'No')
     extract_metadata = True if extract_metadata.lower() == 'yes' else False
+    file_folder = request.POST.get('file_folder', None)
+    if file_folder is not None:
+        if file_folder == "data/contents":
+            file_folder = None
+        elif file_folder.startswith("data/contents/"):
+            file_folder = file_folder.replace("data/contents/", "", 1)
```
Comment (hyi) [inline hs_core/views/__init__.py]: @mauriel133 It is better to replace line 107 with ```file_folder = file_folder[len("data/contents/"):]```

**Comment by pkdash** [comment]
Good suggestion.

File: hs_core/views/__init__.py (hunk 1 above)
Comment (mauriel133) [inline hs_core/views/__init__.py]: Done.

**Comment by hyi** [comment]
@mauriel133 +1 to merge after it passes Jenkins check.

**Pull request merged**